src/client/pages/Section/DataTable/Table/GridRow/RowData/CellHeader/CellHeader.scss

```diff
@@ -28,3 +27,17 @@
     font-weight: 400;
   }
 }
+
+@include rtl {
+  .table-grid__data-cell.header {
+    &.subcategory1 {
+      padding-left: unset;
+      padding-right: 20px;
+    }
+
+    &.subcategory2 {
+      padding-left: unset;
+      padding-right: $spacing-m;
+    }
+  }
+}
```

Comment:
Fixes this:
In english original:
![image](https://github.com/user-attachments/assets/728bf6b7-f19f-49da-97e8-d99a4c38c55e)

How it currently is in Review platform in Arabic:
![image](https://github.com/user-attachments/assets/56551418-c26b-4d67-b590-c6ebaa224697)

With the fix:
![image](https://github.com/user-attachments/assets/e8511403-02b4-43ff-9570-63a329772a30)

now it does match the alignment in English, just flipped.